x-pack/plugins/cases/common/types/api/configure/v1.test.ts

```diff
@@ -596,6 +603,24 @@ describe('configure', () => {
       ).toContain('The length of the description is too long. The maximum length is 1000.');
     });
 
+    it(`throws an error when there are more than ${MAX_TAGS_PER_TEMPLATE} tags`, async () => {
+      const tags = Array(MAX_TAGS_PER_TEMPLATE + 1).fill('foobar');
+
+      expect(
+        PathReporter.report(TemplateConfigurationRt.decode({ ...defaultRequest, tags }))
+      ).toContain(
+        `The length of the field template's tags is too long. Array must be of length <= 10.`
+      );
+    });
+
+    it(`throws an error when the a tag is more than ${MAX_TEMPLATE_TAG_LENGTH} characters`, async () => {
```

Comment:
Could you add one to guarantee it throws an error when one of the tags is an empty string?

Reply:
Fixed in [acbd8ef](https://github.com/elastic/kibana/pull/183743/commits/acbd8efb5826cb0b99c06d7b85466b7030e759dd)